examples/jsm/postprocessing/SSAARenderPass.js

```diff
@@ -7,7 +7,7 @@ import {
 	UniformsUtils,
 	WebGLRenderTarget
 } from '../../../build/three.module.js';
-import { Pass, FullScreenQuad } from '../postprocessing/Pass.js';
```

Comment:
This path is actually a leftover from the time when modules were generated from `examples/js`. It seems almost all post processing pass classes use `../postprocessing/Pass.js;`right now.

Do you mind updating the other classes in `examples/jsm/postprocessing`, too?

Reply:
If you don't want to update the other passes, that's fine. We can still merge the PR and I do a clean up afterwards 👍 .

Reply:
just this file😂,Thank you for your reply. I found that many files have this path problem. It is not very convenient for me to deal with the content of the file code, but it does not affect the operation at all。

Reply:
#22832